components/Feedback/Feedback.tsx

```diff
@@ -88,14 +88,14 @@ export default function PageWithJSbasedForm(props) {
               alt="thumbs-up"
               height="27"
               width="27"
-              onClick={() => handleFeedbackClick("yes")}
+              onClick={() => handleFeedbackClick("true")}
```

Comment:
should it be true and false without the quotes? currently it would be a string

Reply:
Yes, that is correct. I ran into some other Y/N boolen issues further downstream.. but after debugging I think the issue could be related to Inkeep sending data to posthog. I'm going to wait until Inkeep has been upgraded and I'll keep an eye on it.

Reply:
Is there a way to reproduce the issue with a test? Maybe set up a local webserver to handle the Posthog traffic?